### ISSUE TITLE
Workflow doesn't run [Template]

### ISSUE THREAD

**Issue by jonathan-hase** [Mara-Li/obsidian-mkdocs-publisher-plugin#4]
Dear Mara-Li,

I tried everything but the moment I publish a note I get "Error publishing to mt-2022". 

I followed your step-by-step instructions

1. I have github-pro (education) with a repo from your template at https://github.com/jonathan-hase/mt-2022
2. I installed your obsidian plugin and provided my github-name, my repo name and the api-key
3. I changed the website url inside mkdocs.yml to https://jonathan-hase.github.io/mt-2022/
4. I added the `share: true` frontmatter to the notes I want to publish. I tried to publish them.

What do you mean by "Also, don't forget to delete the documentation folder, the contents in assets and clean the notes folder!" inside the README of the template-repo? And do I have to enable the github-page option inside my repo-settings? I am confused, because you write "The blog will be published through [GitHub Page](https://pages.github.com/) using the gh-page branch. Everything is already configured by the template for that.".

Thanks for your time and help!

**Comment by Mara-Li** [comment]
Hello ! 
The error from the plugin is not linked with Github page or the configuration in your mkdocs.yml.

So ! Let's investigate that.

First, could you please provide the contents of your note ? 
Secondly, when trying to Share the note, can you Share with me the error ?
To get the error, open the dev console with CTRL + SHIFT + I.

**Comment by Mara-Li** [comment]
Also, to activate the github page, go to settings (of the repo)→ pages.
Here, check if every thing is correctly configured. You need to have the gh-pages in the branch for the docs !

**Comment by jonathan-hase** [comment]
Hi Mara-Li,

I created a clean gh-pages branch with this commands on github desktop:
```
git checkout --orphan gh-pages
git rm -rf --dry-run .
git rm -rf .
echo "My GitHub Page" > index.html
git add .
git commit -a -m "First pages commit"
git push origin gh-page```
The result is a clean github-pages webpage at https://jonathan-hase.github.io/mt-2022/

The content of my note README.md is
```
---
share: true
---

Testpage
```
(How is the startpage declared? I guess by calling it README or index... or am I wrong?)

The error inside the console is
![image](https://user-images.githubusercontent.com/103068494/166142589-e95a5734-19c5-4020-8784-8eef9e877832.png)

So I checked it out and saw a stupid error: a space in front of my username ... \*headdesk\*
![image](https://user-images.githubusercontent.com/103068494/166142720-4476a3b0-fe2b-4396-95bb-c0e8eb28a4b2.png)

After I corrected the error, the note got sucessfully uploaded. How do I have to do next in order to get the note displayed on mit github-pages website?

Thanks so much for your help!!
- Jonathan

**Comment by Mara-Li** [comment]
Normally, the file is put un your source folder. 
In the workflows folder there is a  file named `ci` that run the conversion to mkdocs. This conversion will create a new file in `notes` folder.

To activate the build gh-pages, check if the branch is correctly setup in your github repo settings, as bellow : 

![image](https://user-images.githubusercontent.com/30244939/166154130-51a39965-f72b-4bbd-81e9-309de6498bfb.png)

(don't worry about the custom domain btw)

Also I will update the plugin to trim any Whitespace in the settings, preveting future problem !

**Comment by jonathan-hase** [comment]
Thanks for adding it to the to-do's :) The thing with the github-action... I hoped the action would be triggered each time I push my notes from inside obsidian into the source folder (with your plugin). But it was not. So I tried to start it manually and I got the message:
![image](https://user-images.githubusercontent.com/103068494/166156359-b30ba878-989b-4321-af66-e889598ff1d9.png)
But then nothing happens.

Also i checked my pages setting and everything is as it should be:
![image](https://user-images.githubusercontent.com/103068494/166156406-3a214e16-276a-4c22-baad-e2d526f9c752.png)

What am I'm doing wrong? o.o

**Comment by Mara-Li** [comment]
It's pretty strange, as the workflow must be triggered by the plugin… Also, it's seems that the mkdocs never be run.
I will investigate on my side using a fork of your blog, checking what happened. I will update you asap when I found the error

**Comment by Mara-Li** [comment]
It's strange because, on my side, any workflow run without problem. For example, in your repo, mkdocs build never run or it needs to after the first edit of mkdocs.yml

**Comment by jonathan-hase** [comment]
I am sorry, that I am of no help here. Somehow the actions are not triggered by push. Super random... I followed each step of your instruction https://github.com/Mara-Li/obsidian-mkdocs-publisher-plugin/#quick-installation-tutorial

**Comment by Mara-Li** [comment]
No worry ! It's okay !
Could You check `Setting → Actions → General` and check if workflow have the permission of reading and writing ?
![image](https://user-images.githubusercontent.com/30244939/166160312-97004d72-45dc-4863-90b3-47f7e54c6bdd.png)

**Comment by jonathan-hase** [comment]
Okay, I changed the permissions. After using Template or Fork the action permissions are set to Read-only and there is no permission in "Allow GitHub Actions to approve pull requests". So I changed that.

I was curious if a fork of your template changes something.  https://github.com/jonathan-hase/fork-test

After I changed the permissions in (fork-test) and (mt-2022), I updated the mkdocs.yml and pushed my README.md note. But again nothing happened. Also starting the ci action manually does nothing.

**Comment by Mara-Li** [comment]
I really, really don't understand, as it works on my side ;_;

**Comment by Mara-Li** [comment]
Could you try to force the workflow to run in `Actions` → `ci` ?

**Comment by jonathan-hase** [comment]
Hi Mara-Li, I tried it and it just gives me the banner

![image](https://user-images.githubusercontent.com/103068494/166163718-435d32cc-2ab4-45e2-a27a-b228ccbe9cde.png)

and than nothing happens. This is mysterious..

Just to be shure: I dont need anything other than a github-repo with your template and your plugin for obsidian?

**Comment by Mara-Li** [comment]
Yeah, normally you don't get nothing more. 
I really don't understand why workflow don't run.

Could you try to rename the workflow file ? Like ci-> ci2 or Something like that. 

I check Google and Other helping site and found NOTHING about our problem.

**Comment by jonathan-hase** [comment]
Hi Mara-Li,

in the meanwhile I tested your actions with my second github account and it immediately worked. After checking every setting in my personal settings and the repo settings for the last time, I wrote to the github support. As it seemed, my account was not activated for actions by github:

![image](https://user-images.githubusercontent.com/103068494/168030595-7691068a-e921-420b-b96b-1af49cb02d5a.png)

Everything works like a charm now. Thank you so much for your help!! It was not a bug in your code in the end.
